IdentityCore/tests/integration/MSIDDefaultAccessorSSOIntegrationTests.m

```diff
@@ -377,39 +377,6 @@ - (void)testSaveTokensWithFactory_whenMultiResourceFOCIResponse_savesTokensOnlyT
     XCTAssertEqual([allAccounts count], 0);
 }
 
-
-- (void)testSaveTokensWithFactoryAndBrokerResponse_shouldFailWithUnsupportedError
```

Comment:
Should there be a test instead of this one?

Reply:
added :)